plugins/connection/lxc.py

```diff
@@ -17,7 +17,6 @@
       remote_addr:
         description:
             - Container identifier
-        default: inventory_hostname
```

Comment:
I think the default should stay, since that's also what ansible-core specifies as a default for all its connection plugins that have a `remote_addr` option. We should start removing it there.